src/refsys/coordinates.py

```diff
@@ -0,0 +1,158 @@
+# -*- coding: utf-8 -*-
+#
+#   Spock mining challenge
+#   Copyright (C) 2022  Óscar Criado, Carlos Moreno
+#
+#   This program is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU General Public License as published by
+#   the Free Software Foundation, either version 3 of the License, or
+#   (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public License
+#   along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+#   See LICENSE
+
+"""
+Conversor module
+
+Functions contained:
+    - coe_to_rv
+    - mean_to_eccentric
+"""
+#######################################################################
+# Imports area
+#######################################################################
+
+# Generic / Built-in
+
+
+# Other Libs
+import numpy as np
+
+# Own Libs
+
+
+#######################################################################
+
+
+#######################################################################
+# Constants area
+#######################################################################
+
+# Eccentric anomaly constants
+MAX_ITER_E = 15

```

Comment:
What do you thkink about move all constants to a new file called `constants.py`? We have multiple constants along legacy code and our new code, and I thhink could be a good practice. We can open new issue for this purpose :)

Reply:
Perfect, fine by me. I will create an issue with the task.